sepal_ui/scripts/utils.py

```diff
@@ -399,15 +400,12 @@ def get_app_version(github_url: str, branch="release") -> str:
     """
     repo_owner, repo_name = parse_github_url(github_url)
 
-    version_url = f"https://raw.githubusercontent.com/{repo_owner}/{repo_name}/{branch}/__version__.py"
-
+    version_url = f"https://raw.githubusercontent.com/{repo_owner}/{repo_name}/{branch}/pyproject.toml"
```

Comment:
I would not get the version number from the repository but from the application folder itself. Normally the folder is pulled from a repository and as such embed the .toml file. It ensures that the version displayed is consistent with what is installed on your machine and not what is available in distant repo. 
Also it avoids getting too much bound to github and start passively supporting other VCS like gitlab or azure devOps